client/elements/sc-page-static.html

```diff
@@ -283,6 +283,7 @@ <h3>{{localize('pageNotFound')}}</h3>
       }
 
       _changeView() {
+          this.$.donate.resetForm();
```

Comment:
Will this get called when going from a static page to a non-static page (the suttaplex for example)?

If it doesn't get called, we can also put this function call it a `disconnectedCallback()`.

Reply:
I tested it and it works fine going from the static to the non-static page. I think that was never a problem in the first place anyway.